app/serializers/invite.js

```diff
@@ -1,10 +1,24 @@
+/* jscs:disable requireCamelCaseOrUpperCaseIdentifiers */
 import ApplicationSerializer from 'ghost-admin/serializers/application';
 import EmbeddedRecordsMixin from 'ember-data/serializers/embedded-records-mixin';
 
 export default ApplicationSerializer.extend(EmbeddedRecordsMixin, {
     attrs: {
-        roles: {embedded: 'always'},
+        role: {embedded: 'always'},
```

Comment:
Changing it to `role: {key: 'role_id'}` removes the need for the custom `normalize` and `serialize` functions below.